specs/introduction.md

```diff
@@ -0,0 +1,127 @@
+# Optimistic Ethereum Introduction
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Foundations](#foundations)
+  - [What is Ethereum scalability?](#what-is-ethereum-scalability)
+  - [What is Optimistic Rollup?](#what-is-optimistic-rollup)
```

Comment:
I think "optimistic rollup" should be qualified with "an", ie "what is an optimistic rollup"?